lib/src/endpoints/browse.dart

```diff
@@ -0,0 +1,21 @@
+part of spotify;
+
+class Browse extends EndpointPaging {
+  Browse(SpotifyApiBase api) : super(api);
+
+  @override
+  String get _path => 'v1/browse';
+
+  /// Returns the new releases.
+  ///
+  /// [country] - a country: an ISO 3166-1 alpha-2 country code. Provide this
+  /// parameter if you want to narrow the list of returned new releases to those
+  /// relevant to a particular country. If omitted, the returned items will be
+  /// globally relevant.
+  Pages<AlbumSimple> getNewReleasess({String? country}) {
```

Comment:
You've been visited by a snake 😸 

```suggestion
  Pages<AlbumSimple> getNewReleases({String? country}) {
```

Reply:
I was a student at Slytherin... 🤷